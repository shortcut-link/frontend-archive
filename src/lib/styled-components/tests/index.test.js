import React from 'react';
import { WithTag } from '../index';

describe('should wrap in an element and pass to children, props', () => {
  it('default tag', () => {
    expect(WithTag.defaultProps.tagName).toEqual('div');
  });

  it('default tag', () => {
    const children = <p>Hello</p>;
    const props = { one: () => {}, foo: 'bar' };

    const func = WithTag({ tagName: 'header', children, props });

    expect(func).toMatchSnapshot();
  });
});
