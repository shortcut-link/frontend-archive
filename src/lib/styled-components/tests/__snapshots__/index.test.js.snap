// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`must wrap in an item with custom tag and transfer of props 1`] = `
<div
  className="one"
  tagName="header"
>
  <div>
    Hello
  </div>
</div>
`;

exports[`must wrap in an item with default tag 1`] = `
<div>
  <div>
    Hello
  </div>
</div>
`;
